### DISCUSSION TITLE
Table cell rowspan bug

### DISCUSSION THREAD

**Discussion by flaribbit** [typst/typst#4116] [General]
### Description

The cell with "38" is not spanning

![image](https://github.com/typst/typst/assets/24885181/a325e0de-8876-494b-a59d-42dd47f58efc)

but when I add another column, it renders correctly

![image](https://github.com/typst/typst/assets/24885181/2ad426e1-fed2-4bca-84b0-1a899bb0a9a8)

### Reproduction URL

https://typst.app/project/rLGSS9FPnddEqEqguLWOX2

### Operating system

Web app, Windows

### Typst version

- [X] I am using the latest version of Typst

**Comment by PgBiel** [comment]
This is correct and expected. The problem is that the second row, as it is `auto` (its height is determined based on cells), has height zero, because the only cells occupying it are rowspans, and:
1. The "38" cell already fits in the first row, so it doesn't need any extra height from the second row.
2. The "row2" cell occupies the second and third rows, so it causes the third row to expand, because rowspans only affect the height of the last spanned auto row.
3. There are no other cells in the second row.

You can solve this by assigning a fixed height to your rows, e.g. with `rows: 2em`.

**Reply by flaribbit** [reply]
Thanks for your detailed explanation! I realize what's going on now.
